### ISSUE TITLE
Bugs about "\\" and "<"

### ISSUE THREAD

**Issue by shanhaiying** [jfbercher/jupyter_latex_envs#3]
`
\begin{proof}

$i<j$

\end{proof}

$i<j$

\begin{proof}

$i< j$

\end{proof}

\hspace{2cm}$\begin{aligned}
r_{i}=  & r_{j}, \;\; i<j   \
a_{i}=  & 2^{\alpha_{i}} r_{i}, \;\;i=1,2,\cdots,n+1\
a_{j}=  & 2^{\alpha_{j}} r_{j}, \;\;j=1,2,\cdots,n+1\ 
\end{aligned}$

\begin{proof}

\hspace{2cm}$\begin{aligned}
r_{i}=  & r_{j}, \;\; i< j   \
a_{i}=  & 2^{\alpha_{i}} r_{i}, \;\;i=1,2,\cdots,n+1\
a_{j}=  & 2^{\alpha_{j}} r_{j}, \;\;j=1,2,\cdots,n+1\ 
\end{aligned}$

\end{proof}
`

![image](https://cloud.githubusercontent.com/assets/1595652/18226615/5f613718-7240-11e6-9c26-67564ae4c126.png)

After deleting the support for \ for line breaks,  we will get the correct result:

![image](https://cloud.githubusercontent.com/assets/1595652/18226642/fe375c14-7240-11e6-912f-0735157b3602.png)

I added support for \par for line breaks.

**Comment by jfbercher** [comment]
Yes, you are right. 
- I made a mistake regarding the replacement of < and > in environments with a blank line. I will revert that.  (Normally we should not use environments with a blank line inside because they are not processed by the notebook's mathjaxutils)
- For the linebreaks, I did not realized that the replacement will also extend to real math environments and mess things inside. Since I do not know a simple way to detect what is real maths (and interpreted after by mathjax ), I will suppress that too. Adding support for \par  is a good idea. Adopted. I will make these changes.
